packages/client/src/provider/in-memory-provider/variant-not-found-error.ts

```diff
@@ -0,0 +1,15 @@
+import { ErrorCode, OpenFeatureError } from '@openfeature/core';
+
+/**
+ * A custom error for the in-memory provider.
+ * Indicates the resolved or default variant doesn't exist.
+ */
+export class VariantFoundError extends OpenFeatureError {
```

Comment:
```suggestion
export class VariantNotFoundError extends OpenFeatureError {
```

Reply:
This will probably need renaming in the server provider as well.